geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/text/TextTools.scala

```diff
@@ -0,0 +1,38 @@
+package org.locationtech.geomesa.utils.text
+
+import org.joda.time.Period
+import org.joda.time.format.PeriodFormatterBuilder
+
+object TextTools {
+  val PeriodFormatter =
+    new PeriodFormatterBuilder().minimumPrintedDigits(2).printZeroAlways()
+      .appendHours().appendSeparator(":").appendMinutes().appendSeparator(":").appendSeconds().toFormatter
+
+  def getPlural(i: Long, base: String): String = getPlural(i, base, s"${base}s")
+
+  def getPlural(i: Long, base: String, pluralBase: String): String = if (i == 1) s"$i $base" else s"$i ${pluralBase}"
+
+  /**
+   * Gets elapsed time as a string
+   */
+  def getTime(start: Long): String = PeriodFormatter.print(new Period(System.currentTimeMillis() - start))
+
+  /**
+   * Gets status as a string
+   */
+  def getStatInfo(successes: Long, failures: Long): String = {
```

Comment:
I might move this somewhere else since its not a generic text tool but is specific to ingest.